manifest.yaml

```diff
@@ -39,3 +39,14 @@ processors:
 connectors:
   - gomod: go.opentelemetry.io/collector/connector/forwardconnector v0.99.0
   - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/connector/spanmetricsconnector v0.99.0
+
+providers:
+  - gomod: go.opentelemetry.io/collector/confmap/provider/envprovider v0.99.0
+  - gomod: go.opentelemetry.io/collector/confmap/provider/fileprovider v0.99.0
+  - gomod: go.opentelemetry.io/collector/confmap/provider/httpprovider v0.99.0
+  - gomod: go.opentelemetry.io/collector/confmap/provider/httpsprovider v0.99.0
+  - gomod: go.opentelemetry.io/collector/confmap/provider/yamlprovider v0.99.0
+  - gomod: github.com/Dynatrace/dynatrace-otel-collector/confmap/provider/eecprovider v0.4.0
```

Comment:
What about calling this `v0.0.0` since there's not an actual version involved here? 🤔

Reply:
That's a fair point considering the replace statement. Done.